ChangeLog.rst

```diff
@@ -9,6 +9,94 @@ Unreleased
   releasing next version, the "Unreleased" header will be replaced with
   appropriate version header and this help text will be removed.
 
+Version 2.0.0
+-------------
+
+Admin
+~~~~~
+
+- Add basic Manufacturer views (914d456)
+- Basic Supplier management (65787f3, 3953e24, b864e27)
+- Add image field for category (9157b26)
+- Fix issue with price not being saved (eb90ff5)
+- Optionally disable creating shops (f427f6a)
+- Add `Product Media` tab to product editing (279a26d)
+- Tabify translated fields (46b49c5)
+- Variation UI styles (bf5a9ba)
+- Media Browser Rehaul (9f4ff68, b7b3d39, 35e4354, 1d22009, 4c4345f)
+- Add `ImageChoiceWidget` (7548fb5)
+- Admin: Actually send `data-filter` to media browser from browse widgets (cca4a57)
+- Admin form error indicators (b13a4c3)
+- Add keyboard shortcuts to megasearch (d5c75dc)
+- Show first language tab with errors (a3c1fd0)
+- Update styles for admin form error indicators (00c644d)
+- Show errors more clearly (7a8620b)
+- Make folder clicks work at media browser (d66954b)
+- Fix paths of generated source maps (3a738ce)
+- Add styles for bootstrap input-group (f61a0b2)
+- Time interval attribute now renders as `DecimalField` (45e774f)
+- Notify: Add better error handling for "step edit"-popup (df76c12, 2366ff1)
+- Better variation error handling (a877781)
+- Fix issue where visibility errors caused an error (acda42f)
+- Update admin category view (a404fa0)
+- Admin datetimepicker (f1681e5)
+
+Core
+~~~~
+
+- Rework `SimplePricing` and pricing in general (eaf40f3, 5cbeb44)
+- Add non-ASCII support for supplier name (2c56b7e)
+- Fix checking of duplicate settings (685d393)
+- Add new fields to shop core (d56dd24)
+- Add `DiscountPricingModule` (SHOOP-1289) (f9f3789)
+- Fix PriceInfo usage with non-one quantities (SHOOP-1462) (1bb1c02)
+- Fix shoop.core.migrations.0006 (b605d04)
+- Tax clean-up and refactoring (SHOOP-975) (2cb5a50, 182daff)
+- Prevent `Shop` being deleted when image was deleted (c0edb08)
+
+Frontend
+~~~~~~~~
```

Comment:
Are you referring to "shoop.front"? I thought we've named it "Shoop Front"... maybe this subtitle should be just "Front"?